docs/contracts.rst

```diff
@@ -115,7 +115,7 @@ This means that cyclic creation dependencies are impossible.
         {
             // Check some arbitrary condition.
             address tokenAddress = msg.sender;
-            return (keccak256(abi.encodePacked(newOwner)) & 0xff) == (bytes20(tokenAddress) & 0xff);
+            return keccak256(abi.encodePacked(newOwner))[0] == bytes20(tokenAddress)[0];
```

Comment:
@chriseth Is this better? This is now not more complicated than before IMO, but we might still want to choose something entirely different that's even simpler.

Reply:
Ah no, this won't be the same...